test/Rings/mpolyquo-localizations.jl

```diff
@@ -552,7 +552,7 @@ end
   U = complement_of_point_ideal(R, [0, 0])
   L, _ = localization(A, U)
   AA, iso, iso_inv = simplify(L)
-  @test all(x==iso(iso_inv(x)) for x in gens(A))
-  @test all(x==iso_inv(iso(x)) for x in gens(AA))
+  @test all(x==iso(iso_inv(x)) for x in gens(AA))
+  @test all(x==iso_inv(iso(x)) for x in gens(A))
```

Comment:
Observation: The isomorphisms are defined between L and AA, not between A and AA.